NitroxClient/MonoBehaviours/AnimationController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+
+namespace NitroxClient.MonoBehaviours
+{
+    public class AnimationController : MonoBehaviour
+    {
+        public static readonly float SMOOTHING_SPEED = 0.95f;
```

Comment:
Correct me if I'm wrong, if the other player initiated a player animation 65ms in the past, are there an instances were it is prudent to subtract that from the duration to keep the clients in sync?